Bottom: 70a882dcff42e9b1e2de7bb1f1185554222c4111
Top:    0dbbf1a9ae9d915ec521c6509739bce5b864a66a
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-25 14:24:39 +0800

utofs-5.0.7 - remove nosymlink pseudo option


---

diff --git a/modules/mount_nfs.c b/modules/mount_nfs.c
index bbbb1de..e61320b 100644
--- a/modules/mount_nfs.c
+++ b/modules/mount_nfs.c
@@ -125,6 +125,10 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name, int
 
 			o_len = end - cp + 1;
 			if (strncmp("nosymlink", cp, o_len) == 0) {
+				warn(ap->logopt, MODPREFIX
+				     "the \"nosymlink\" option is depricated "
+				     "and will soon be removed, "
+				     "use the \"nobind\" option instead");
 				nosymlink = 1;
 			} else if (strncmp("nobind", cp, o_len) == 0) {
 				nobind = 1;
